redis/src/aio/async_std.rs

```diff
@@ -162,7 +163,12 @@ impl RedisRuntime for AsyncStd {
                 .danger_accept_invalid_hostnames(true)
                 .use_sni(false)
         } else {
-            TlsConnector::new()
+            let mut tls_connector = TlsConnector::new();
+            if let Some(ca_cert) = ca_cert {
+                let cert = native_tls::Certificate::from_der(ca_cert)?;
```

Comment:
Should this be 'from_der' or 'from_pem'?

Reply:
Perhaps `ca_cert` should be an `Option<Certificate>`, since I don't think the redis library needs to care about the cert format.

Reply:
I updated to `Option<Certificate>` I had to wrap the `native_tls::Certificate` because it didn't implement Eq, PartialEq, Debug required by `ConnectionAddr`